runners_manager/runner/RunnerManager.py

```diff
@@ -64,23 +65,33 @@ def update(self, github_runners: list[dict]):
         for elem in github_runners:
             if elem['name'] in self.runners:
                 runner = self.runners[elem['name']]
-                runner.update_status(elem)
-
+                if elem['status'] == 'online' and elem['busy'] is True:
+                    status = 'running'
+                else:
+                    status = elem['status']
+                runner.update_status(status)
```

Comment:
I don't like the idea  that a `Runner` state is calculated out of it

Reply:
Me neither, I can add a function in the runner that can take the status from github and do this calculation inside the runner

Reply:
I tried solving this issue, checkout the code and let me know what you think